addons/binding/org.openhab.binding.geofence/README.md

```diff
@@ -0,0 +1,160 @@
+# Geofence Binding
+
+This binding allows you to connect mobile GPS tracking applications to an openHAB installation and detect 
```

Comment:
Please add a line break after every sentence throughout this file. This makes diffing easier and has no impact on the rendering.